units/host_termination.go

```diff
@@ -137,16 +139,20 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 
 	// clear the running task of the host in case one has been assigned.
 	if j.host.RunningTask != "" {
-		grip.Warning(message.Fields{
-			"message":  "Host has running task; clearing before terminating",
-			"job":      j.ID(),
-			"job_type": j.Type().Name,
-			"host":     j.host.Id,
-			"provider": j.host.Distro.Provider,
-			"task":     j.host.RunningTask,
-		})
+		if j.TerminateIfBusy {
+			grip.Warning(message.Fields{
+				"message":  "Host has running task; clearing before terminating",
+				"job":      j.ID(),
+				"job_type": j.Type().Name,
+				"host":     j.host.Id,
+				"provider": j.host.Distro.Provider,
+				"task":     j.host.RunningTask,
+			})
 
-		j.AddError(model.ClearAndResetStrandedTask(j.host))
+			j.AddError(model.ClearAndResetStrandedTask(j.host))
```

Comment:
should we return early if this errors?

Reply:
No we should still terminate the host regardless